satpy/etc/readers/seviri_l2_bufr.yaml

```diff
@@ -1147,23 +1232,25 @@ datasets:
 
   qual:
```

Comment:
Please add `flag_values` and `flag_meanings`.

Reply:
done